src/txorphanage.h

```diff
@@ -72,11 +72,19 @@ class TxOrphanage {
         return m_orphans.size();
     }
 
-protected:
-    struct OrphanTx {
+    /**
+     * Allows providing orphan information externally
+     */
+    struct OrphanTxBase {
         CTransactionRef tx;
         NodeId fromPeer;
         NodeSeconds nTimeExpire;
+    };
+
+    std::vector<OrphanTxBase> GetOrphanTransactions();
```

Comment:
Do you think defining this as a const function `std::vector<OrphanTxBase> GetOrphanTransactions() const;` makes sense? I do not think it changes any members of the `TxOrphanage`.

Reply:
Thanks! Will include this in next push.

Reply:
Updated